File: play-book-api/organization-service/src/main/java/org/atlas/requests/SpaceRequest.java
package org.atlas.requests;

import org.atlas.enums.SpaceVisibility;

import java.util.List;
import java.util.UUID;

public record SpaceRequest(String name, String description, String icon, UUID team_id, SpaceVisibility visibility, List<SpaceMemberRequest> members  ) {
}

File: play-book-ui/src/app/core/requests/space.request.ts
import {SpaceVisibilityEnum} from '../../store/models/enums/space-visibility.enum';

import {SpaceMembersRequest} from './space-members.request';

export interface SpaceRequest {
  name: string;
  description: string;
 icon: string;
 team_id: string;
 members: SpaceMembersRequest[];
 visibility: SpaceVisibilityEnum;

}
